lib/widgets/card_property.dart

```diff
@@ -1,27 +1,59 @@
 import 'package:flutter/material.dart';
+import 'package:hearthstoneapp/utils/constants.dart';
 
 class CardProperty extends StatelessWidget {
   final double fontSize;
   final Color color;
+  final String propertyName;
   final String text;
   final FontWeight weight;
+  final bool visibility;
 
   const CardProperty({
     Key? key,
-    this.fontSize = 20,
+    this.fontSize = Constants.mediumFontSize,
     this.color = Colors.white,
     required this.text,
+    required this.propertyName,
     this.weight = FontWeight.normal,
+    this.visibility = true,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Text(
-      text,
-      style: TextStyle(
-        fontSize: fontSize,
-        color: color,
-        fontWeight: weight,
+    return Visibility(
+      visible: visibility,
+      child: Padding(
+        padding: Constants.verticalSmallPadding,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Flexible(
+              flex: Constants.flex2,
```

Comment:
We dont use number to name constants, here you can change it instead to flex2 like cardPropertyFlex or something similar

Reply:
Changed them to propertyCardSmall, ...Medium and ...Large respectively

Reply:
To be able to use the values in more places I created a CustomFlex class with the constant values for small, medium and large.